packages/next/build/webpack-config.ts

```diff
@@ -732,6 +732,12 @@ export default async function getBaseWebpackConfig(
 
   const emacsLockfilePattern = '**/.#*'
 
+  // Allowing importing TS/TSX files from outside of the root dir.
+  let externalDir
+  if (config.experimental.externalDir) {
+    externalDir = path.join(dir, config.experimental.externalDir)
```

Comment:
This should likely not be a configurable directory, instead the `include` should just be removed when the flag is enabled.